documentation/docs/20-core-concepts/20-load.md

```diff
@@ -90,9 +90,8 @@ export async function load() {
 </script>
 
 <main>
-	<slot>
-		<!-- +page.svelte is rendered here -->
-	</slot>
+	<!-- +page.svelte is rendered here -->
```

Comment:
you could arguably say "in this slot" instead of "here" in case that helps with the original confusion at all

Reply:
```suggestion
	<!-- +page.svelte is rendered in this <slot> -->
```